gocr: add page (#12294)
tldr-pages/tldr: pages/common/gocr.md

# gocr

> Optical Character Recognition tool.

> > Recognize characters using its engine, and prompt the user for unknown patterns to store them in a daabase.

Reviewer: ```suggestion
> Recognize characters using its engine, and prompt the user for unknown patterns to store them in a database.
```
No idea why Codespell didn't catch it.

> More information: <https://manned.org/gocr.1>.

> - Recognize characters in the [i]nput image and [o]utput it in the given file. Put the database ([p]) in `./existing/db/folder/` (verify that the folder exists or DB usage will silently be skipped). [m]ode 130 means create + use + extend database:

Reviewer 2: ```suggestion
- Recognize characters in the [i]nput image and [o]utput it in the given file. Put the database ([p]) in `path/to/db_directory` (verify that the folder exists or DB usage will silently be skipped). [m]ode 130 means create + use + extend database:
```

> - Recognize characters in the [i]nput image and [o]utput it in the given file. Put the database ([p]) in {{./existing/db/folder/}} (verify that the folder exists or db usage will silently be skipped). [m]ode 130 means create + use + extend database:

Reviewer: ```suggestion
- Recognize characters in the [i]nput image and [o]utput it in the given file. Put the database ([p]) in `./existing/db/folder/` (verify that the folder exists or DB usage will silently be skipped). [m]ode 130 means create + use + extend database:
```

> `gocr -m 130 -p {{path/to/db_folder/}} -i {{path/to/input_image.png}} -o {{path/to/output_file.txt}}`

Reviewer 2: ```suggestion
`gocr -m 130 -p {{path/to/db_directory}} -i {{path/to/input_image.png}} -o {{path/to/output_file.txt}}`
```

> - Recognize characters, and assume all characters are numbers [C]:

Reviewer: ```suggestion
- Recognize characters, and assume all [C]haracters are numbers:
```
How about this?

Author: Oh yeah sorry, I missed it

> `gocr -m 130 -p {{./existing/db/folder/}} -i {{input/image.png}} -o {{output/file.txt}} -C "0-9"`

Author: I will apply this, but keep the final / on the db folder, because gocr says it is mandatory

> `gocr -m 130 -p {{path/to/db_folder/}} -i {{path/to/input_image.png}} -o {{path/to/output_file.txt}} -C "{{0..9}}"`

Reviewer 2: ```suggestion
`gocr -m 130 -p {{path/to/db_directory}} -i {{path/to/input_image.png}} -o {{path/to/output_file.txt}} -C "{{0..9}}"`
```

- Recognize characters with a cert[a]inty of 100% (characters have a higher chance to be treated as unknown):

> `gocr -m 130 -p {{./existing/db/folder/}} -i {{input/image.png}} -o {{output/file.txt}} -a 100`

Reviewer 2: ```suggestion
`gocr -m 130 -p {{path/to/existing_db_folder}} -i {{path/to/input_image.png}} -o {{path/to/output_file.txt}} -a 100`
```
See the comment above.

> `gocr -m 130 -p {{path/to/db_folder/}} -i {{path/to/input_image.png}} -o {{path/to/output_file.txt}} -a 100`

Reviewer 2: ```suggestion
`gocr -m 130 -p {{path/to/db_directory}} -i {{path/to/input_image.png}} -o {{path/to/output_file.txt}} -a 100`
```
